public/index.html

```diff
@@ -3,24 +3,28 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
+<link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Can you indent this so it is in line with the rest of the code, i.e the title, please.